app/models/review.rb

```diff
@@ -0,0 +1,7 @@
+class Review < ActiveRecord::Base
+  belongs_to :user, foreign_key: :user_id
+  belongs_to :video, foreign_key: :video_id
```

Comment:
oh got it. I just like to keeping there for my own reference since it does no harm